geochem_inverse_optimize.py

```diff
@@ -310,7 +328,9 @@ def process_data(
 
 def main():
     results = process_data(
-        data_dir="data/", data_filename="data/geochem_no_dupes.dat", excluded_elements=["Bi", "S"]
+        data_dir="data/",
+        data_filename="data/geochem_no_dupes.dat",
+        excluded_elements=["Bi", "S"],
     )
     print(results)
```

Comment:
When I run main.py I only get 7 elements (Be, Na, Mg, Al, P, K, Ca) outputting results out of the 53 (55 minus 2) complete elemental observations.

Reply:
That's expected from Line 306
```
for element in ELEMENT_LIST[0:20]:
```

Reply:
Got it, thanks!